dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueDnsResolutionWorker.java

```diff
@@ -64,7 +69,7 @@ public void run() {
                 // this logging may be helpful in informing us of problems in the system.
                 log.info("Output refreshable has been garbage collected, no need to continue polling");
             } else {
-                doUpdate(null);
+                updateTimer.time(() -> doUpdate(null));
```

Comment:
whoops, this is the wrong place to put this, i think we need to measure within doUpdate.